src/main/kotlin/no/nav/familie/ef/sak/brev/VedtaksbrevService.kt

```diff
@@ -166,7 +168,9 @@ class VedtaksbrevService(
         }
 
         val beslutterSignatur = if (signaturMedEnhet.skjulBeslutter) "" else signaturMedEnhet.navn
-        return html.replace(BESLUTTER_SIGNATUR_PLACEHOLDER, beslutterSignatur)
+        return html
+            .replace(BESLUTTER_SIGNATUR_PLACEHOLDER, beslutterSignatur)
+            .replace(BESLUTTER_VEDTAKSDATO_PLACEHOLDER, LocalDate.now().norskFormat())
```

Comment:
Valgte å ikke validere denne for å kunne være bakåtkompatibel med eksisterende brev som ikke inneholder placeholder.